wasmcloud-chat/actors/messages/Makefile

```diff
@@ -0,0 +1,34 @@
+COLOR ?= always # Valid COLOR options: {always, auto, never}
+CARGO = cargo --color $(COLOR)
+TARGET = target/wasm32-unknown-unknown
+DEBUG = $(TARGET)/debug
+RELEASE = $(TARGET)/release
+KEYDIR ?= .keys
+VERSION = $(shell cargo metadata --no-deps --format-version 1 | jq -r '.packages[].version')
+
+.PHONY: all build check clean doc test update
+
+all: build
+
+build:
+	@$(CARGO) build	
+	wash claims sign $(DEBUG)/messages.wasm -c wasmcloud:httpserver --name "messages" --ver $(VERSION) --rev 0
+
+check:
+	@$(CARGO) check
+
+clean:
+	@$(CARGO) clean
+
+doc:
+	@$(CARGO) doc
+
+test: build
+	@$(CARGO) test
+
+update:
+	@$(CARGO) update
+
+release:
+	@$(CARGO) build --release	
+	wash claims sign $(RELEASE)/messages.wasm -c wasmcloud:httpserver --name "messages" --ver $(VERSION) --rev 0
```

Comment:
same messages comment ad the `build`section

Reply:
We'll be filling out the actual claims as we build the actors. In addition to `wasmcloud:messaging`, I'm guessing that this actor is also going to need `wasmcloud:eventstreams` at the very least.